### DISCUSSION TITLE
Vue 3 and Nuxt 3 support.

### DISCUSSION THREAD

**Discussion by devbipu** [vidstack/player#847] [General]
Hi there, 
I'm a Vue.js developer. I'm working on a video-sharing platform. I want to use this player in my Vue/Nuxt3 project. But I didn't find any documentation on Vue 3. I found React documentation. Can I use it on my Nuxt 3 project? 

Best regards
devbipu

**Comment by WaiserHai** [comment]
no answer ?

**Comment by mihar-22** [comment]
Sorry missed this somehow. We have a big release dropping soon and I've built examples using the library with Vue. There'll be installation docs for it too! You can have an early preview of it here on the `next` branch: https://github.com/vidstack/examples/tree/main/player/vue

**Reply by devbipu** [reply]
I hope it will be the best video player. I'm waiting to use it in my project. Thank you :)

**Comment by mihar-22** [comment]
Just an update here that it's now available. Updated the examples link above, and you can get started with our [Vue installation docs](https://www.vidstack.io/docs/player/getting-started/installation/vue?provider=video&styling=css). We also ship JSX types for it too so you have full syntax highlighting and props/events. 

See [1.0@next release notes](https://github.com/vidstack/vidstack/discussions/935) for more information.

**Reply by devbipu** [reply]
Great announcement. Thank you :)
